rules/windows/process_creation/proc_creation_win_disable_service.yml

```diff
@@ -0,0 +1,37 @@
+title: Sc Execution to Disable Services
```

Comment:
Title say "sc" but can trigger on a powershell "Set-Service".
Somethink like "Sc Execution or Powershell Set-Service to Disable Services"

Reply:
Thanks, will fix this